src/ButterCMS/ButterCMS.php

```diff
@@ -24,18 +27,23 @@ public function __construct($authToken)
     {
         $this->authToken = $authToken;
 
-        $this->client = new Client();
+        $this->client = new Client([
+            'headers' => [
+                'X-Butter-Client' => 'PHP/2.3.0',
+            ],
```

Comment:
Is trailing `,` a syntax error?

Reply:
Yup! I often do it because then if you add items to the array the diff is a lot cleaner as it only shows the new entry as having changed.

Reply:
Nope! I often do it because then if you add items to the array the diff is a lot cleaner as it only shows the new entry as having changed.